src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends FigureColor {
```

Comment:
Circle **is not** Color. Circle **IS A** Figure
```suggestion
public class Circle extends Figure {
```